server/src/main/java/org/apache/iotdb/db/metadata/mtree/ConfigMTree.java

```diff
@@ -460,19 +458,19 @@ protected void transferToResult(IMNode node) {
    */
   public Pair<Set<String>, Set<PartialPath>> getChildNodeNameInNextLevel(PartialPath pathPattern)
```

Comment:
Is this method still in use in business? If TestOnly, eliminate it.

Reply:
It seems deprecated. I have deleted this method and related test.